src/Bundle/Resources/config/services.xml

```diff
@@ -22,12 +22,16 @@
     <services>
         <defaults public="true" />
 
-        <service id="sylius.grid.data_extractor.property_access" class="Sylius\Component\Grid\DataExtractor\PropertyAccessDataExtractor">
+        <service id="Sylius\Component\Grid\DataExtractor\PropertyAccessDataExtractor">
             <argument type="service" id="property_accessor" />
         </service>
+        <service id="sylius.grid.data_extractor.property_access" alias="Sylius\Component\Grid\DataExtractor\PropertyAccessDataExtractor" />
+
         <service id="sylius.grid.array_to_definition_converter" class="Sylius\Component\Grid\Definition\ArrayToDefinitionConverter">
             <argument type="service" id="event_dispatcher" />
         </service>
+        <service id="Sylius\Component\Grid\Definition\ArrayToDefinitionConverterInterface" alias="sylius.grid.array_to_definition_converter" />
```

Comment:
Why do you flip the alias when there's an interface?

Reply:
We need to keep the previous id and add the new service id with the interface.

Reply:
Yeah, I get that, the thing I had in mind is why not go
```
<service id="Sylius\Component\Grid\Definition\ArrayToDefinitionConverterInterface" class="Sylius\Component\Grid\Definition\ArrayToDefinitionConverter">
    <argument type="service" id="event_dispatcher" />
</service>
<service id="sylius.grid.array_to_definition_converter" alias="Sylius\Component\Grid\Definition\ArrayToDefinitionConverterInterface" />
```
so the new `id` is always in the service definition and the old service name is aliased.

Reply:
@NoResponseMate @lchrusciel done